webpack.config.js

```diff
@@ -12,33 +12,51 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+const path = require('path');
+
 module.exports = {
-  entry: './src/browser.js',
+  entry: './src/index.ts',
   output: {
-    library: 'tasks',
-    filename: './tasks.js',
+    library: 'CloudTasks',
```

Comment:
breaking change, need to use `--main-service tasks`

Reply:
got it! thanks